docs/source/models/concept_models/vivarium_mncnh_portfolio/mncnh_interventions.rst

```diff
@@ -0,0 +1,130 @@
+    - Decrease multiplicatively
+    - Yes
+    - 
+
+Baseline Coverage Data
+++++++++++++++++++++++++
+
+The :ref:`MNCNH Portfolio model <2024_concept_model_vivarium_mncnh_portfolio>` includes data on current coverage levels for the specific locations we are focused on modeling.  We might wish to enhance this by predicting coverage in additional locations, in correlation with other key intervention technologies, based on sources such as existing SPA and SARA data.
+
+Vivarium Modeling Strategy
+--------------------------
+
+We will use the enhanced decision tree below to find the probability of RDS mortality with and without access to CPAP that are logically consistent with the baseline delivery facility rates and baseline CPAP coverage (and population level RDS mortality probability). Since the neonatal model include timesteps for both the early and late neonatal periods, this  alculation must be calculated for both time periods.
+
+We will then add an attribute to each simulant indicating whether the birth occurs at a facility with access to CPAP (which will be dependent on the facility choice, i.e. no access for home births and lower access for BEmONC than CEmONC facilities).
+
+We will then use the conditional probabilities for simulants with and without access to determine which simulants die from RDS.
+
+.. todo::
+
+  include the relative risk of RDS mortality with CPAP here and in the concept model document, without duplicating it (see recent PR to add quickstart docs in the readme for an example of how)
+
+
+Calibration Strategy
+--------------------
+
+The following decision tree shows all of the paths from delivery facility choice to CPAP availability.  Each path corresponds to a disjoint event that we can sum over to find the population probability of RDS mortality.
+
+.. graphviz::
+
+    digraph CPAP {
+        rankdir = LR;
+        start [color="none"]
+        facility [label="1"]
+        home [label="p_RDS_without_CPAP"]
+        BEmONC [label="7"]
+        CEmONC [label="7"]
+        BEmONC_wo [label="p_RDS_without_CPAP"] 
+        BEmONC_w [label="p_RDS_with_CPAP"]
+        CEmONC_wo [label="p_RDS_without_CPAP"] 
+        CEmONC_w [label="p_RDS_with_CPAP"]
+
+        start -> facility  [label = "delivery facility type"]
+        facility -> home  [label = "home birth"]
+        facility -> BEmONC  [label = "BEmONC"]
+        facility -> CEmONC  [label = "CEmONC"]
+
+        BEmONC -> BEmONC_w  [label = "CPAP available"]
+        BEmONC -> BEmONC_wo  [label = "CPAP unavailable"]
+
+        CEmONC -> CEmONC_w  [label = "CPAP available"]
+        CEmONC -> CEmONC_wo  [label = "CPAP unavailable"]
+    }
+
+.. math::
+    \begin{align*}
+        p_\text{RDS} 
+        &= \sum_{\text{paths without CPAP}} p(\text{path})\cdot p_\text{RDS_wo}\\
+        &+ \sum_{\text{paths with CPAP}} p(\text{path})\cdot p_\text{RDS_w}\\[.1in]
+        p_\text{RDS_w} &= RR_\text{CPAP} \cdot p_\text{RDS_wo}
+    \end{align*}
```

Comment:
You should define your variables somewhere. Does $p_\text{RDS}$ mean the probability of dying from RDS? Is $p_\text{RDS-wo}$ the probability of dying from RDS if you don't get CPAP and $p_\text{RDS-w}$ the probability of dying of RDS if you do get CPAP? Are those probabilities conditional on having RDS or not?

The notation is a bit sloppy since it doesn't match the graph exactly, but maybe that's not a big deal. But `p_\text{RDS_w}` is malformed $LaTeX$ since underscores can only be used in math mode (GitHub gave me an error when I tried to render it).

Also, it would probably be good to explain how to compute $p(\text{path})$. It looks like the edges of your graph are events, not probabilities, right?

Reply:
I'll add more detail, thanks for calling this out.  It looks ok in my local html, though:
<img width="268" alt="image" src="https://github.com/user-attachments/assets/5e762c7e-84a0-43b9-9607-b586a9a1c7ad" />